src/components/Discover/TrendingTokens.tsx

```diff
@@ -415,19 +405,19 @@ function TrendingTokenRow({ item }: { item: TrendingTokensType['trendingTokens']
 
             <View style={{ gap: 12, marginLeft: 'auto' }}>
               <View style={{ flexDirection: 'row', gap: 2, alignItems: 'center' }}>
-                <Text color={isPositiveChange ? 'green' : 'red'} size="11pt" weight="bold">
-                  {isPositiveChange ? '􀄨' : '􀄩'}
+                <Text color={token.priceChange.timeframe > 0 ? 'green' : 'red'} size="11pt" weight="bold">
```

Comment:
we should break this condition out and define it above since we're relying on it so much

```
const isPositiveTrend = token.priceChange.timeframe > 0
```

Reply:
Begs the question of what do we show if the trend **IS** 0? Because I don't think green or red satisfy that condition

Reply:
very unlikely be will return a trending token with 0 price change, but yeah a grey zero?

Reply:
probably safe to do like tertiary or something yeah